docs/installation.rst

```diff
@@ -30,7 +30,7 @@ for running NengoLoihi,
 as well as for running models
 using the NxSDK directly.
 
-Note, you *must* use Python 3.5.2 when working with NxSDK.
+Note, you *must* use Python 3.8 when working with NxSDK.
```

Comment:
I'm not sure if this is actually a requirement. I'm using 3.7 and it seems to work fine. I think our hardware tests actually use 3.6.

Reply:
Oh nice, that simplifies things. I know they don't have .pyc file anymore so that makes sense :+1: